src/string/JWT.ts

```diff
@@ -113,3 +117,21 @@ export const Type: SchemableParams1<t.URI> = pipe(t.string, t.refine(isJWT, 'JWT
  * @category Instances
  */
 export const Encoder: SchemableParams2<Enc.URI> = Enc.id()
+
+/**
+ * Note: This will produce unsigned JWTs, with `alg: "none"`.
+ *
+ * @since 0.0.3
+ * @category Instances
+ */
+export const Arbitrary: SchemableParams1<Arb.URI> = fc.object().map(json => {
+  const header = base64Encode(JSON.stringify({ alg: 'none', typ: 'JWT' })).replaceAll(
+    /[=+/]/g,
+    c => (c === '/' ? '_' : c === '+' ? '-' : c === '=' ? '' : c)
+  )
+  const payload = base64Encode(JSON.stringify(json)).replaceAll(/[=+/]/g, c =>
+    c === '/' ? '_' : c === '+' ? '-' : c === '=' ? '' : c
+  )
+  const signature = ''
+  return [header, payload, signature].join('.')
+}) as Arb.Arbitrary<JWT>
```

Comment:
_really_ wanted these to be legit JWTs but I can't find a decent cross-platform crypto library that isn't async